lib/Parser/Parse.cpp

```diff
@@ -6318,28 +6318,36 @@ ParseNodePtr Parser::GenerateEmptyConstructor(bool extends)
     ParseNodePtr pnodeFncSave = m_currentNodeFunc;
     m_currentNodeFunc = pnodeFnc;
 
+    ParseNodePtr argsId = nullptr;
+    ParseNodePtr *lastNodeRef = nullptr;
     ParseNodePtr pnodeBlock = StartParseBlock<buildAST>(PnodeBlockType::Parameter, ScopeType_Parameter);
-    ParseNodePtr pnodeInnerBlock = StartParseBlock<buildAST>(PnodeBlockType::Function, ScopeType_FunctionBody);
-    pnodeBlock->sxBlock.pnodeScopes = pnodeInnerBlock;
-    pnodeFnc->sxFnc.pnodeBodyScope = pnodeInnerBlock;
-    pnodeFnc->sxFnc.pnodeScopes = pnodeBlock;
 
-    ParseNodePtr *lastNodeRef = nullptr;
     if (extends)
```

Comment:
Added comment.

---
In reply to: [70012406](https://github.com/Microsoft/ChakraCore/pull/1255#discussion_r70012406) [](ancestors = 70012406)